app/Helper/Constant.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Helper;
+
+class Constant
+{
+    CONST NOT_DELETED = 0;
+    CONST SOFT_DELETE = 1;
+    CONST HARD_DELETE = 9;
+}
```

Comment:
I think this is very old approach before PHP introduces enum class.

https://enversanli.medium.com/how-to-use-enums-with-laravel-9-d18f1ee35b56

What I would suggest, create an Enums folder. and move these to there.

```php
enum DeleteStatus: string
{
    case NOT_DELETED = 0;
    case SOFT_DELETE = 1;
    case HARD_DELETE = 9;
}
```


And then use like - 
```php
DeleteStatus::SOFT_DELETE
```

Reply:
solved